src/mocks/data.ts

```diff
@@ -123,7 +113,7 @@ export const datasetSummary = {
   accession: 'GHGAD588887987',
   description:
     'Test dataset for Metadata Repository get dataset details call. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Vel risus commodo viverra maecenas accumsan lacus vel. Sit amet risus nullam eget felis eget nunc lobortis mattis. Iaculis at erat pellentesque adipiscing commodo. Volutpat consequat mauris nunc congue. At lectus urna duis convallis convallis tellus id interdum velit. Gravida cum sociis natoque penatibus et. Mauris in aliquam sem fringilla ut morbi. Ultrices gravida dictum fusce ut. At consectetur lorem donec massa sapien faucibus et molestie.',
-  type: ['Test Type'],
+  type: ['Test Type', 'Test Type 2'],
```

Comment:
I think this is now called types instead of type in the backend. I fixed this but you should double check.